docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Setting up the Okta Service as a hosted integration
+description: How to set up the Okta Service as a hosted integration
+---
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- An Okta organization with an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token) created.
+
+<Details title="Okta API token permissions">
+(!docs/pages/includes/okta-permissions.mdx hosted="true"!)
+</Details>
+
+### Enrolling a hosted Okta Integration
+
+
+#### The Enrolment Process
+
+Enrolling the Okta hosted integration will automatically perform the following
+actions:
+
+ 1. Create an Okta SAML Application called `Teleport $clustername` in the
+    upstream Okta organization
+ 1. Assign the Okta `Everyone` group to the new SAML Application
+ 1. Download the IdP Metadata for the new SAML Application
+ 1. Create a new SAML Authentication Connector called `okta-integration` in the
+    Teleport cluster
+ 1. Configure the Okta integration to draw its user pool from the Okta users
+    assigned to the new `Teleport $clustername` Okta SAML application
+ 1. Start the hosted Okta integration, including the Okta synchronization
+    process
+
+If Teleport Identity is enabled, the enrolment process will also guide you
+through the process of enabling SCIM provisioning for the Okta SAML application.
+
+<Admonition type="note">
+If a SAML Connector called `okta-integration` already exists, the enrolment
+process will attempt to adopt this connector and use the Okta SAML Application
+it points to, rather than create a new one.
+
+This adoption process will fail if the SAML Connector was created with Teleport 
+versions prior to 15.0.0, or was created manually. In this case, the best course
+of action is to either delete or rename the existing SAML Connector and let the
+integration enrolment create a new one.
+</Admonition>
+
+#### How to enrol
```

Comment:
Can we sequence the enrolment process by steps,  like 
step 1/4 Configuring SCIM provisioning
step 2. Enable SCIM Provisioning
step 4/4 ...